cpp/include/raft/sparse/convert/csr.cuh

```diff
@@ -102,6 +105,41 @@ void adj_to_csr(raft::resources const& handle,
   detail::adj_to_csr(handle, adj, row_ind, num_rows, num_cols, tmp, out_col_ind);
 }
 
+/**
+ * @brief Converts a bitmap matrix into unsorted CSR format matrix.
+ *
+ * @tparam       bitmap_t    Underlying type of the bitmap.
+ * @tparam       index_t     Indexing type used.
+ * @tparam       value_t     Data type of CSR
+ * @tparam       nnz_t       Type of CSR
+ *
+ * @param[in]    handle      RAFT handle
+ * @param[in]    bitmap      input raft::bitmap_view
+ * @param[out]   csr         output raft::device_csr_matrix_view
+ */
+template <typename bitmap_t, typename index_t, typename value_t, typename nnz_t>
+void bitmap_to_csr(raft::resources const& handle,
+                   raft::core::bitmap_view<bitmap_t, index_t> bitmap,
+                   raft::device_csr_matrix_view<value_t, index_t, index_t, nnz_t> csr)
+{
+  auto csr_view = csr.structure_view();
+
+  RAFT_EXPECTS(bitmap.get_n_rows() == csr_view.get_n_rows(),
+               "Number of rows in bitmap must be equal to "
+               "number of rows in csr");
+
+  RAFT_EXPECTS(bitmap.get_n_cols() == csr_view.get_n_cols(),
+               "Number of columns in bitmap must be equal to "
+               "number of columns in csr");
+
+  detail::bitmap_to_csr(handle,
```

Comment:
I agree with you on checking 'nnz', but it can hurt the performance; I intended to recommend the caller to check `nnz` when building the `bitset` before calling. Do you think this is okay?